src/extension/app.css

```diff
@@ -65,6 +65,25 @@
   --hlx-sk-special-view-table-border: rgba(0,0,0, 0.5);
   --hlx-sk-palette-title-bg: rgba(0 0 0 / 10%);
   --hlx-sk-user-bg: #c0c0c0;
+  --hlx-sk-badge-color-white: #fff;
+  --hlx-sk-badge-color-black: #000;
+  --hlx-sk-badge-bg-default: rgba(112, 112, 112);
+  --hlx-sk-badge-bg-gray: rgba(209, 209, 209);
+  --hlx-sk-badge-bg-red: rgba(246, 88, 67);
+  --hlx-sk-badge-bg-orange: rgba(244, 129, 12);
```

Comment:
For this to be spectrum compliant there should be dark and light versions of each color.

@rofe should we support that here or just leave it for the next-gen extension?

Reply:
I'd say next-gen